src/GitPrompt.ps1

```diff
@@ -128,11 +130,18 @@ if (Get-Module NuGet) {
 }
 
 function Write-Prompt($Object, $ForegroundColor, $BackgroundColor = -1) {
+    if ($GitPromptSettings.AnsiConsole) {
+        $e = [char]27 + "["
+        $f = Get-ForegroundVirtualTerminalSequence $ForegroundColor
+        $b = Get-BackgroundVirtualTerminalSequence $BackgroundColor
+        return "${f}${b}${Object}${e}0m"
```

Comment:
We need to be able to handle terms that support 24-bit color like the upcoming Windows 10 CU drop.  See my Twitter post on that - https://twitter.com/r_keith_hill/statuses/835556870697930753  This was something I was encapsulating in a Color class in my prototype.